src/content/plugins/split-chunks-plugin.md

```diff
@@ -0,0 +1,220 @@
+---
+title: SplitChunksPlugin
+contributors:
+  - sokra
+  - jeremenichelli
+related:
+  - title: "webpack 4: Code Splitting, chunk graph and the splitChunks optimization"
+    url: https://medium.com/webpack/webpack-4-code-splitting-chunk-graph-and-the-splitchunks-optimization-be739a861366
+---
+
+Originally in webpack, chunks (and modules import inside them) were connected by a parent-child relationship in the internal webpack graph.
+
+This connection didn't allow to further optimizations and reluted into more coded downloaded.
+
+webpack v4 removes the `CommonsChunkPlugin` in favor of `optimization.splitChunks` and `optimization.runtimeChunk` options. Here is how the new flow works.
+
+## Defaults
+
+Out of the box `SplitChunksPlugin` should work great for most users.
+
+By default the plugin only affect on-demand chunks, because changing initial chunks would affect the script tags that the HTML file should include to run the project.
```

Comment:
`affect` should be `affects`

Reply:
@blowery I don't agree, affect is correct grammar

Reply:
@audiolion, are you sure? I think if "plugin" were plural, then I agree "affect" should be used (ie "the plugins only affect on-demand chunks"), however given it's singular, "affects" seems more correct to me.

Reply:
haha, I was reading the wrong "affect", `initial chunks would affect(s) the script tags...`

Reply:
Lol, same happened to me. Thanks.